ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -101,7 +101,7 @@ wan_router:
             - name: Ethernet1
               wan_carrier: Inmrasat
               wan_circuit_id: S511
-              dhcp_accept_default_route: true
+              dhcp_accept_default_route: false
               ip_address: dhcp
             - name: Ethernet2
               wan_carrier: AWS-1
```

Comment:
Remove `dhcp_accept_default_route: true` from Ethernet2, to test that default value is respected.
Currently we have no test case for it.

Reply:
we do in the other host `node-type-l3-interfaces`

Reply:
Yup I see the updated molecule now.
Thanks